README.md

```diff
@@ -1,5 +1,5 @@
 # sharp-cli
-> CLI for [sharp](https://www.npmjs.com/package/sharp).
+> CLI for [sharp](https://www.npmjs.com/package/sharp) a high performance Node.js image processing module.
```

Comment:
Could you add a comma before your changes?

Reply:
👍